eth/eventhandler/event_handler.go

```diff
@@ -424,46 +458,53 @@ func (eh *EventHandler) processLocalEvent(txn basedb.Txn, event localevents.Even
 	case OperatorAdded:
 		data := event.Data.(contract.ContractOperatorAdded)
 		if err := eh.handleOperatorAdded(txn, &data); err != nil {
-			return fmt.Errorf("handle OperatorAdded: %w", err)
+			return fmt.Errorf("handle %s: %w", event.Name, err)
```

Comment:
honestly I would keep this PR clean and not do this here, or at least have it in separate commit.